src/dags/anpr.py

```diff
@@ -70,42 +65,69 @@ def import_csv_data(*args, **kwargs):
             try:
                 hook.run(sql)
             except Exception as e:
-                raise Exception("Failed to create data: {}".format(str(e)[0:150]))
-            print("Created {} recods".format(len(items)))
+                raise Exception(f"Failed to create data: {e}"[:150])
+            logger.debug("Created %d records.", len(items))
```

Comment:
Do debug logs end up anywhere useful?

Reply:
Yes, Airflow captures the logs per task run. You can download them from the GUI

Reply:
But this is just converting a print statement to a logging statement.